src/nerve/compose.py

```
@@ -190,18 +190,47 @@ async def create_anthropic_proxy(
         AnthropicProxyConfig,
         AnthropicProxyServer,
     )
 
+    # Load config file if specified
+    file_config: dict[str, Any] = {}
+    config_path = config_file or os.environ.get("NERVE_PROXY_CONFIG")
+    if config_path:
+        try:
+            import yaml
+
+            with open(config_path) as f:
```

Ruff (0.14.10)

Async functions should not open files with blocking methods like `open`

(ASYNC230)